src/cocaine-app/planner/__init__.py

```diff
@@ -73,12 +78,86 @@ def __init__(self, db, niu, job_processor, namespaces_settings):
                     self._couple_defrag
                 )
 
+            if self.params.get('ttl_cleanup', {}).get('enabled', False):
```

Comment:
Fixed in https://github.com/yandex/mastermind/pull/311/commits/3e401778f178741610800a49b28e8245e296de80